im.pidgin.Pidgin3.yml

```diff
@@ -0,0 +1,128 @@
+app-id: im.pidgin.Pidgin3
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: pidgin3
+finish-args:
+  # X11 + XShm access
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  # Wayland access
+  - "--socket=wayland"
+  # GPU acceleration if needed
+  - "--device=dri"
+  # Needs to talk to the network:
+  - "--share=network"
+  # Audio
+  - "--socket=pulseaudio"
+  # Notifications (purple-toast)
+  - "--talk-name=org.freedesktop.Notifications"
+  # Accessibility
+  - "--talk-name=org.a11y.Bus"
+  # libsecret plugin
+  - "--talk-name=org.freedesktop.secrets"
+  # kwallet plugin
+  - "--talk-name=org.kde.kwalletd5"
+  - "--talk-name=org.kde.kwalletd6"
+  # Bonjour
+  - "--system-talk-name=org.freedesktop.Avahi"
+  # Plugin Paths
+  - "--env=PURPLE_PLUGIN_PATH=/app/plugins/purple-3"
+  - "--env=PIDGIN_PLUGIN_PATH=/app/plugins/pidgin-3"
+
+add-extensions:
+  im.pidgin.Purple3.Plugin:
+    directory: plugins
+    version: default
+    add-ld-path: lib
+    merge-dirs: purple-3
+    no-autodownload: true
+    autodelete: true
+    subdirectories: true
+  im.pidgin.Pidgin3.Plugin:
+    directory: plugins
+    version: default
```

Comment:
you can omit the `version` field, it is only useful if there is going to be api/abi breaks and you are going to maintain more than 1 parallel branches.

"default" doesn't sound like it needs any of that.